lib/smart_answer_flows/energy-grants-calculator/outcomes/_warm_home_discount.govspeak.erb

```diff
@@ -1 +1 @@
-[Warm Home Discount scheme](/the-warm-home-discount-scheme) - £<%= calculator.warm_home_discount_amount %> discount on your energy bills
+- [Warm Home Discount scheme](/the-warm-home-discount-scheme) - £140 discount on your energy bills
```

Comment:
Fair enough. @pmanrubia predicted list items like that will be difficult to keep.